schutzbot/deploy.sh

```diff
@@ -34,16 +34,17 @@ for RETRY in {1..10}; do
 done
 
 # Pull image-builder image
+ARCH=$(uname -m)
 QUAY_REPO_URL="quay.io/osbuild/image-builder-test"
 QUAY_REPO_TAG="${CI_PIPELINE_ID}"
-sudo podman pull --creds "${V2_QUAY_USERNAME}":"${V2_QUAY_PASSWORD}" "${QUAY_REPO_URL}":"${QUAY_REPO_TAG}"
+sudo podman pull --creds "${V2_QUAY_USERNAME}":"${V2_QUAY_PASSWORD}" "${QUAY_REPO_URL}":"$ARCH"-"${QUAY_REPO_TAG}"
```

Comment:
I just wonder if we want to put the `$ARCH` as a postfix instead of a prefix. Sounds a bit more natural... What do you think?

Reply:
This is better, I agree.